Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: container_build container_tag acr_login container_push
+
+ENGINE := docker
+CONTAINER_NAME := pyrenew-hew
+CONTAINER_REMOTE_NAME := $(ACR_TAG_PREFIX)$(CONTAINER_NAME)":latest"
```

Comment:
I would add this as optional perhaps? See for instance: https://github.com/CDCgov/cfa-epinow2-pipeline/blob/338f0ec947084f5201b68d1be7c55bd0169e07d0/Makefile#L2-L4

Reply:
Azure CR expects the engine to be named docker (I'm actually using an aliased podman here)